### PR TITLE
[Rest Server] Add label for gpu in docker container

### PR DESCRIPTION
**Pull request by abuccts**
Add label for gpu in docker container.

### CODE REVIEW

File: rest-server/src/templates/yarnContainerScript.mustache [hunk 1]
```diff
@@ -85,6 +89,7 @@ docker run --name $docker_name \
   --security-opt apparmor:unconfined \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
+  --label GPU_NO=$gpu_no \
```
Comment (YanjieGao) [inline rest-server/src/templates/yarnContainerScript.mustache]: suggest follow previous style. refine GPU_NO to PAI_TASK_GPU_IDS

Comment (abuccts) [reply rest-server/src/templates/yarnContainerScript.mustache]: PAI only sees gpu from 0 to `PAI_GPU_NUM - 1`, don't you need gpu on host?

Comment (YanjieGao) [reply rest-server/src/templates/yarnContainerScript.mustache]: Only will use this docker label gpu id and job/task info to join with custom exporter's gpu metrics. Have question. exporter will use nvidia-smi's gpu id series, need confirm with you docker label's id is consist with nvidia-smi's gpu id sequence?

Comment (abuccts) [reply rest-server/src/templates/yarnContainerScript.mustache]: `nvidia-smi` from host or docker get different results. `GPU_NO` is for gpus on the host.

File: rest-server/src/templates/yarnContainerScript.mustache [hunk 2]
```diff
@@ -48,16 +48,20 @@ while /bin/true; do
   [ -f "/tmp/pai-root/alive/$APP_ID/docker_$CONTAINER_ID" ] && [ ! "$(docker ps | grep $docker_name)" ] && break
 done &
 
+gpu_no=''
 nvidia_devices=''
 {{# taskData.gpuNumber }}
 nvidia_devices+='--device=/dev/nvidiactl --device=/dev/nvidia-uvm'
 gpu_bitmap=$(perl -e 'printf "%b",'$CONTAINER_GPUS)
```
Comment (sterowang) [inline rest-server/src/templates/yarnContainerScript.mustache]: >CONTAINER_GPUS [](start = 36, length = 14)

Oops, how about simply export $CONTAINER_GPUS directly?

**Pull request merged**